scripts/commands/pos.lua

```diff
@@ -68,7 +68,7 @@ function onTrigger(player, arg1, arg2, arg3, arg4, arg5)
 
     -- report or move position
     if x == nil or y == nil or z == nil then
-        player:PrintToPlayer(string.format("%s's position: X %.4f  Y %.4f  Z %.4f", targ:getName(), targ:getXPos(), targ:getYPos(), targ:getZPos()))
+        player:PrintToPlayer(string.format("%s's position: X %.4f  Y %.4f  Z %.4f  Rot %i", targ:getName(), targ:getXPos(), targ:getYPos(), targ:getZPos(), targ:getRotPos()))
```

Comment:
Looks like you're just reporting here, was there an intent to also add an optional parameter to send as well?  Fine either way, but could easily be added as well.

Reply:
then intention was to report only, as its helpful in creating dynamic entities and other things. i can't really see a reason why i'd want to send a player to a specific rotation

Reply:
I think we'd actually have problems adding rot as input, because we would normally have it before the zone param - and then all our existing uses would need updated and thats a huge pain point for NPC scripts with pos in the header (which is there because of the MASSIVE convenience it provides)